source/manual/architecture-shallow-dive.html.md

```diff
@@ -0,0 +1,145 @@
+---
+owner_slack: "#govuk-developers"
+title: Architectural summary of GOV.UK
+section: Applications
+type: learn
+layout: manual_layout
+parent: "/manual.html"
+---
+
+The following content is a summary of the architecture of GOV.UK.
+
+This content is a less detailed version of the [Architectural deep-dive of GOV.UK](/manual/architecture-deep-dive.html), and is for both developers and non-developers to refer to.
+
+The content summarises the architecture by stating what happens when a user:
+
+- visits a page on GOV.UK
+- publishes content to GOV.UK
+
+As this is for both a developer and non-developer audience, the content contains [definitions](#definitions) at the end.
+
+## A user visits a page on GOV.UK
+
+This content will go through 3 scenarios where a user:
+
+- visits the GOV.UK homepage
+- visits a GOV.UK webpage
+- searches GOV.UK using the search page
+
+### A user visits the GOV.UK homepage
+
+The user visits https://www.gov.uk or https://gov.uk to get to the GOV.UK hpmepage.
+
+The user's browser uses [DNS](#dns) to figure out where to send the request, and sends that request to the Fastly content delivery network (CDN).
+
+[Fastly](https://www.fastly.com/) is a cloud computing services provider and content delivery network (CDN).
```

Comment:
Now we no longer have a Definitions section, there's nowhere for this to link to. I can't find a GOV.UK resource that describes DNS, so we could link to a third-party site (e.g. [Cloudflare](https://www.cloudflare.com/en-gb/learning/dns/what-is-dns/)) or attempt to summarise DNS inline, e.g.:

```suggestion
The user's browser uses DNS (Domain Name System) to look up which IP address is associated with the domain. We've configured this to be the IP address for [Fastly](https://www.fastly.com/), which is the CDN (content delivery network) that GOV.UK uses. 
```